### DISCUSSION TITLE
[Support]: Searching for "Bark" labe for last 7 Days Returns No Tracked Objects Found

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#15404] [General Support]
### Describe the problem you are having

I have a bunch of bark detects that happen each day. I am trying to search for them outside of filtering by camera, and then choosing events and bark for example. When I use search by "label" bark it says No Tracked Objects Found. Not sure why this is.

### Version

beta-15-2

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
NA
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Hikvision Cameras

**Comment by NickM-27** [comment]
This has been fixed for the next beta
